src/java/org/apache/cassandra/tools/nodetool/RelocateSSTables.java

```diff
@@ -20,26 +20,38 @@
 import java.util.ArrayList;
 import java.util.List;
 
-import io.airlift.airline.Arguments;
-import io.airlift.airline.Command;
-import io.airlift.airline.Option;
 import org.apache.cassandra.tools.NodeProbe;
-import org.apache.cassandra.tools.NodeTool;
+import org.apache.cassandra.tools.nodetool.layout.CassandraUsage;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+import picocli.CommandLine.Parameters;
+
+import static org.apache.cassandra.tools.nodetool.CommandUtils.parseOptionalKeyspace;
+import static org.apache.cassandra.tools.nodetool.CommandUtils.parseOptionalTables;
+import static org.apache.cassandra.tools.nodetool.CommandUtils.concatArgs;
 
 @Command(name = "relocatesstables", description = "Relocates sstables to the correct disk")
-public class RelocateSSTables extends NodeTool.NodeToolCmd
+public class RelocateSSTables extends AbstractCommand
 {
-    @Arguments(usage = "<keyspace> <table>", description = "The keyspace and table name")
+    @CassandraUsage(usage = "<keyspace> <table>", description = "The keyspace and table name")
     private List<String> args = new ArrayList<>();
 
-    @Option(title = "jobs",
-            name = {"-j", "--jobs"},
+    @Parameters(index = "0", description = "The keyspace name", arity = "0..1")
+    private String keyspace;
+
+    @Parameters(index = "1", description = "The table name", arity = "0..1")
```

Comment:
did the old implementation support only single table to pass?

Reply:
Fixed.